portfolio/src/main/webapp/Gallery.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <style>
+        div.gallery{
+            margin:5px;
+            border:1px solid #ccc;
+            float: left;
+            width:auto;
+        }
+        div.gallery:hover(
+            border:1px solid #777;
+        )
+        div.gallery img {
+            width: 100%;
+            height: auto;
+        }
+        div.desc {
+            padding: 15px;
+            text-align: center
+        }
+
+    </style>
```

Comment:
Normally, styles like this are defined in a css file. Could you move this there?